gravitee-apim-console-webui/src/services-ngx/task.service.ts

```diff
@@ -13,35 +13,31 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import { IHttpPromise } from 'angular';
-
-import UserService from './user.service';
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Inject, Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
 
 import { Constants } from '../entities/Constants';
 import { PagedResult } from '../entities/pagedResult';
-import { User } from '../entities/user';
+import { Task } from '../entities/task/task';
 
-class TaskService {
-  constructor(private readonly $http: ng.IHttpService, private readonly Constants: Constants, private readonly UserService: UserService) {
-    'ngInject';
-  }
+@Injectable({
+  providedIn: 'root',
+})
+export class TaskService {
+  constructor(private readonly http: HttpClient, @Inject('Constants') private readonly constants: Constants) {}
 
   getTaskSchedulerInSeconds(): number {
```

Comment:
I wonder if we should do a getTasksWithAutoFetch or something like this 🤔 WDYT ? 
to avoid this get and have only subscribe in component ?